### ISSUE TITLE
how can use the attritute scores?  

### ISSUE THREAD

**Issue by ranjiewwen** [aimerykong/deepImageAestheticsAnalysis#21]
i use the mobilenetv2 fine tune aadb and get regression six class attritute scores and total scores .
the result : 
`SROCC_v: 0.484	
SROCC_v: 0.404	
SROCC_v: 0.616	
SROCC_v: 0.455	
SROCC_v: 0.162	
SROCC_v: 0.607 total scores	`
 
we can see the total scores maybe normal , but the attritute scores is bad . how can improve the attribute scores??? maybe the data is not good ,because the attribute label only [-1,0,1] . i use all the data(include the scores is 0.00) , can you give me some advise to improve the attribute scores ?thanks !

- "The supplemental material provides additional details about
dataset collection and statistics of the resulting data" where is the  supplemental material ? i want to read more.

**Comment by aimerykong** [comment]
As AADB is sort of small in scale, in our experiment, we first fine-tune the AlexNet backbone over AADB for aesthetic rating. Then we freeze AlexNet backbone and train only the new branch for attribute classification. Attribute annotated with '0' value may be dominant, so it is possible to re-weigh '-1' and '1' (though we didn't do this in our work).

In one word, we freeze backbone and train new branch (2 or 3 new layers) to avoid overfitting. Moreover, as our main goal is aesthetic rating, attribute prediction acts only as auxiliary signal such that it is possible the prediction looks merely OKay in practice.

Our arxiv version contains the supplementary material as the appendix. (https://arxiv.org/pdf/1606.01621.pdf)
